auth_token/backends.py

```diff
@@ -14,16 +14,19 @@ def authenticate(self, device_uuid=None, mobile_login_token=None, **kwargs):
         if not device_uuid or not mobile_login_token:
             return None
 
-        device_key = DeviceKey.objects.filter(uuid=device_uuid, is_active=True).first()
-        if not device_key:
+        device_key_qs = DeviceKey.objects.filter(uuid=device_uuid, is_active=True)
```

Comment:
more users can have same device key

Reply:
Does not make much sense to me... like when 2 users use the same phone?

Reply:
according to Jarda 2 users can be logged on same device, because you can use profiles. The second situation is when you remove application and install it with new user. The old token cannot be removed becaus new user has no permission to do that